assets/css/reboot.css

```diff
@@ -2277,6 +2277,18 @@ tr, td, th {
     background-color: #006292;
     color: #fff;
 }
+.btn--bluesky {
+    background-color: #007bb6;
+    color: #fff;
+}
+.btn--bluesky:visited {
+    background-color: #007bb6;
+    color: #fff;
+}
+.btn--bluesky:hover {
+    background-color: #006292;
```

Comment:
In the preview, I'm not seeing this color change in any meaningful way when hovering over any of the footer icons. Maybe an issue to track for another PR?

Reply:
Actually to be a bit more specific, I don't see these color changes in the live site either.

Reply:
> Actually to be a bit more specific, I don't see these color changes in the live site either.

Hrm.  I'm not sure what's going on with that.  I was just mimicking the existing buttons.